internal/collector/pgadmin_test.go

```diff
@@ -44,7 +50,7 @@ collector.yaml: |
   extensions:
     file_storage/pgadmin_data_logs:
       create_directory: false
-      directory: `+pgadmin.LogDirectoryAbsolutePath+`/receiver
+      directory: /var/lib/pgadmin/logs/receiver
```

Comment:
👍🏻 I recall assert diffs being harder to read when these long strings aren't constant.